hydrolib/core/dflowfm/ini/models.py

```diff
@@ -55,6 +56,15 @@ class Config:
         extra = Extra.ignore
         arbitrary_types_allowed = False
 
+    def __init__(self, **data):
```

Comment:
Was there a reason we could not put this in a `root_validator`?

Reply:
I am not sure, but I think it had to do with the exclude fields not yet being initialized, thus always returning an empty list for it, will check this.

Reply:
I think we can move it to the root validator, but the root validator is run after the init, so moving it might be in conflict with your statement below.

Reply:
Okay, let's leave it in the init.